README.md

```diff
@@ -88,6 +88,11 @@ into regions according to script, as defined in ISO15924.
 This annotation describes the sentences of the text. By convention, the sentences cover the entire text.
 Whitespace at the end of each sentences is incorporated into that sentence.
 
+### DATA_TYPE_REGION ###
+
+This annotation describes separates the text into regions of structured
+and unstructured text.
```

Comment:
What do “structured” and “unstructured” mean?

Reply:
@keren171 how do you want to define "structured" vs "unstructured"?